cvat-ui/src/components/task-page/validation-report.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
```

Comment:
NB: I don't see any existing code in Cvat use React. Are you comfortable adding this dependency?

Reply:
I didn't add React to the project, it was used by maintainers already. Tasks list page and task detail page are written in React, but annotation UI is not  (old annotation UI which we are using, the new one is written in React I believe).